packages/ra-ui-materialui/src/list/DatagridLoading.js

```diff
@@ -72,10 +72,10 @@ export default ({
                             className={classes.expandIconCell}
                         >
                             <IconButton
+                                // TODO: add an aria-label
                                 className={classes.expandIcon}
                                 component="div"
                                 aria-hidden="true"
-                                role="expand"
```

Comment:
why?

Reply:
Because after looking for it, it appears to not be an existing aria role.

lint rule: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/aria-role.md
list of valid roles: https://www.w3.org/TR/wai-aria/#role_definitions

Reply:
Besides, this button does nothing. It's juste here for placeholder UI while loading

Reply:
gotcha